src/client/common/client_common.cpp

```diff
@@ -140,11 +141,13 @@ std::shared_ptr<grpc::Channel> mp::client::make_channel(const std::string& serve
         const std::vector<QString> cert_dirs{data_location + gui_client_cert_dir, data_location + cli_client_cert_dir};
         for (const auto& cert_dir : cert_dirs)
         {
+            mpl::log(mpl::Level::trace, "client", fmt::format("Searching for client certs in {}…", cert_dir));
             if (client_certs_exist(cert_dir))
             {
                 if (auto rpc_channel{
                         create_channel_and_validate(server_address, get_ssl_credentials_opts_from(cert_dir))})
                 {
+                    mpl::log(mpl::Level::trace, "client", "found!");
```

Comment:
If this is to stay, it should probably be more expressive.

Reply:
Looking at what I added here again, this will print out after this line I added: https://github.com/canonical/multipass/blob/623e8fe3bdb598292fc480c13a1496db3eee63e1/src/client/common/client_common.cpp#L144

I guess I can make it more verbose, but basically it is just confirming that it was found for the previous log message.

Reply:
Sure, problem is when these two lines get separated by some other concurrent logging, it's difficult to correlate. There's more cases like that of course, but we should try and avoid adding more :)

Reply:
Ok, done.